experimental/devices/pca9685/pca9685_test.go

```diff
@@ -0,0 +1,95 @@
+package pca9685
```

Comment:
Please include copyright header.

Reply:
added in https://github.com/google/periph/pull/437/commits/a682bff12501065e21d972c8a9dc029f46777a36